docs/src/pages/[platform]/connected-components/authenticator/quick-start-install.android.mdx

```diff
@@ -0,0 +1,24 @@
+The Authenticator component is built using [Jetpack Compose](https://developer.android.com/jetpack/compose). Enable Jetpack Compose by adding the following to the `android` section of your **app**'s `build.gradle` file:
+
+```groovy
+buildFeatures {
+    compose true
+}
+composeOptions {
+   kotlinCompilerExtensionVersion '1.2.0'
+}
+```
+
+Add the following dependencies to your **app**'s `build.gradle` file and click "Sync Now" when prompted:
+
+```groovy
+dependencies {
+
+    // Amplify Auth dependency (optional if using your own credentials provider)
+    // TODO : update amplify dependency to latest
+    implementation 'com.amplifyframework:aws-auth-cognito:2.5.0'
```

Comment:
Should the note about the auth dependency being optional be removed? The TODO can be removed and auth dependency updated to 2.7.1.